src/Microsoft.DotNet.Watcher.Tools/Internal/Implementation/ProjectReader.cs

```diff
@@ -36,11 +34,11 @@ public bool TryReadProject(string projectFile, out IProject project, out string
         }
 
         // Same as TryGetProject but it doesn't throw
```

Comment:
Yeah, i'm not sure what it means either. Looks like the comment has survived since days of dnx-watch. Probably can be cleaned out.

![image](https://cloud.githubusercontent.com/assets/2696087/19396338/862d1c1e-91f7-11e6-8394-4da29666639a.png)